src/App.jsx

```diff
@@ -1,104 +1,28 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getCommentsById(id) {
+  return commentsFromServer.filter(comment => comment.postId === id) || null;
```

Comment:
```suggestion
  return commentsFromServer.filter(comment => comment.postId === id);
```
null is redundant here, because the filter method always returns an array